src/applications/simple-forms/mock-simple-forms-patterns/pages/mockRadioRelationshipToVeteran.js

```diff
@@ -0,0 +1,21 @@
+import {
+  titleUI,
+  titleSchema,
+  relationshipToVeteranUI,
+  relationshipToVeteranSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    wcTitle: titleUI('Web component'),
+    wcv3RelationshipToVeteran: relationshipToVeteranUI(),
```

Comment:
it looks like the test is failing because this key now has mismatch with the tests.